clang/test/CodeGen/ptrauth-cpu-feature.c

```diff
@@ -0,0 +1,59 @@
+// REQUIRES: aarch64-registered-target
```

Comment:
A couple of test cases with `-NOT` suffix for FileCheck directives can be added. Particularly, ensure that with major == 8, minor >= 3 or major >= 9 the feature is enabled and the error message is not produced.

Reply:
Fixed